src/hashtable.py

```diff
@@ -0,0 +1,26 @@
+class HashTable(object):
+
+    def __init__(self, buckets=1024):
+        self.buckets = buckets
+        self.init_table()
+
+    def init_table(self):
+        self._table = []
+        for x in range(self.buckets):
+            self._table.append([])
+
+    def get(self, key):
```

Comment:
If when you `set` or `get` the value for a key, you reduce the hash value modulo the number of buckets, why not just include that modulo operation IN the hash method?